### ISSUE TITLE
Incorrect author in FB OG meta code

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#345]
I was going to share the new Events page https://www.digitalgov.gov/events/ to Facebook but noticed that title and summary are wrong. 

When I looked at the code, I saw that Will Sullivan is listed as author in the head for FB OG (this is site-wide)

**Q**: 
Can JZ/Dan do a mass find & replace to change: 

&lt;meta property="article:author" content="willsullivan" /&gt;

to: 

&lt;meta property="article:author" content="" /&gt; 

so the correct author profile URL(s) or ID(s) can be specified when known?

**Comment by jeremyzilar** [comment]
Yeah, I can work on this now. Thank you for flagging @ToniBonittoGSA

**Comment by jeremyzilar** [comment]
I'd also like to remove the `OG:image` values and let Twitter/Facebook/Slack pick up what ever images they want to in the post.

**Comment by jeremyzilar** [comment]
Reading up on:
https://developers.facebook.com/docs/reference/opengraph/object-type/article/
http://ogp.me/
